lib/src/screens/ionia/cards/ionia_buy_gift_card.dart

```diff
@@ -101,7 +100,7 @@ class IoniaBuyGiftCardPage extends BasePage {
                       prefixIcon: Padding(
                         padding: EdgeInsets.only(
                           top: 5.0,
-                          left: _width / 4,
+                          left: 150,
```

Comment:
please refrain from using exact numbers when centering widgets or so
we can use `min(MediaQuery.of(context).size.width, ResponsiveLayoutUtil.instance.kDesktopMaxWidthConstraint) / 4`